system-container/tmpfiles.template

```diff
@@ -1 +1,2 @@
 d    $VAR_LIB_OPENSHIFT_INSTALLER - - - - -
+f    $VAR_LOG_OPENSHIFT_LOG - - - - -
```

Comment:
This commit can also we squashed, no need to introduce a two-line file in two separate commits.